content/blog/the-ilorest-tool-in-conjunction-with-directly-attached-and-controller-connected-drives.md

```diff
@@ -0,0 +1,717 @@
+---
+title: The iLORest tool in conjunction with Directly Attached and
```

Comment:
Please modify the title to comply with Brand standards request for the use of sentence case. Also, there should not be a period at the end of a title. It should look more like this:

Using the iLORest tool with directly attached and controller-connected drives

By the way, do you really mean "directly attached" or "direct-attached"?
https://en.wikipedia.org/wiki/Direct-attached_storage 

I've always heard it referred to as direct-attached storage...